### ISSUE TITLE
tag.plu.cn plugin added with pull request #669 failed to load

### ISSUE THREAD

**Issue by chhe** [chrippa/livestreamer#672]
I am using the latest nightly build for Windows (livestreamer-v1.11.1-10-ge80fa95-win32.zip) on WIndows 7 x64.

The tag.plu.cn plugin (merged with pull request #669) fails to load with the following message:

```
C:\Program Files (x86)\Livestreamer> http://www.twitch.tv/tsm_bjergsen
Failed to load plugin tga:
  File "c:\program files (x86)\livestreamer\livestreamer-1.11.1-py2.7.egg\livestreamer\plugins\tga.py", line 15, in <module>
    _stream_re = re.compile("<iframe style='(.*)?' frameborder='0' scrolling='no' src='http://v.qq.com/iframe/live_player.html\?cnlid=(?P<cnid>\d+)&width=\d+&height=\d+'></iframe>");
  File "re.py", line 190, in compile
  File "re.py", line 242, in _compile
error: nothing to repeat
```

**Comment by MrAkergren** [comment]
I'm running OS X Mavericks (version 10.9) and also get an error with the tga plugin using the latest version "livestreamer 1.11.1".

       Failed to load plugin tga:
    File "/Library/Python/2.7/site-packages/livestreamer-1.11.1-py2.7.egg/livestreamer/plugins/tga.py", line 15, in <module>
    _stream_re = re.compile("<iframe style='(.*)?' frameborder='0' scrolling='no'     src='http://v.qq.com/iframe/live_player.html\?cnlid=(?P<cnid>\d+)&width=\d+&height=\d+'></iframe>");
    File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/re.py", line 190, in compile
    return _compile(pattern, flags)
    File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/re.py", line 242, in _compile
    raise error, v # invalid expression
    error: nothing to repeat

**Comment by wolftankk** [comment]
hmm, I'm sorry guys.

I test my code on the Python 2.7.9 and Python3, it's ok. Today, my colleague tells me, he has geted the errors with the plugin on python2.7.3.

I'll fix it.

**Comment by chrippa** [comment]
Resolved by bbafd228549103ef194bff9daa8a1d8f3c4e6124.
